src/Features/OnboardingTasks.php

```diff
@@ -394,6 +395,22 @@ public static function add_extended_task_list_item( $new_task_name ) {
 		}
 	}
 
+	/**
+	 * Removes item to the extended task list.
```

Comment:
Removes item *from* the extended task list.

Reply:
Thanks @samueljseay, you're right. This was modified in the commit ddd0fe6